standards/RDA/ruleset.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+<ruleset name="RDA">
+	<description>Reader's Digest Association WordPress Coding Standards</description>
+
+	<rule ref="WordPress-Core">
+		<!-- This rule is foolish and breaks many clean code uses -->
+		<exclude name="PEAR.Functions.FunctionCallSignature.Indent" />
+		<exclude name="WordPress.PHP.YodaConditions" />
+		<exclude name="WordPress.WP.GlobalVariablesOverride" />
```

Comment:
This one can be dangerous. Why would we override WordPress' global variables? Can we review some use cases for this?